File: PRAK601/PRAK601_2310817110011_Ghani-Mudzakir.c
#include <stdio.h>

int main()
{
    int banyak_baris, banyak_kolom;
    // kita akan membuat sebuah int untuk menampung nilai dari banyak nya baris lwn kolom
    scanf("%d %d", &banyak_baris, &banyak_kolom);

    // kita akan membuat array dengan ukuran banyak_baris x banyak_kolom
    int array_wadah [banyak_baris][banyak_kolom];

    // kita akan membuat loop untuk memasukan banyak bilangan ke dalam array 
    for (int i = 0; i < banyak_baris; i++)
    {
        for (int j = 0; j < banyak_kolom; j++)
        {
            scanf("%d", &array_wadah[i][j]);
        }
    }
    for (int i = 0; i < banyak_baris; i++)
    {
        for (int j = 0; j < banyak_kolom; j++)
        {
            printf("%d ", array_wadah[i][j]);
        }
        printf("\n");
    }
    

    return 0;
}

File: PRAK601/PRAK601_2310817110011_Ghani-Mudzakir.py
banyak_baris, banyak_kolom = map(int, input().split())

# Kita akan membuat array dengan ukuran banyak_baris x banyak_kolom
array_wadah = []

# Kita akan membuat loop untuk memasukkan banyak bilangan ke dalam array dalam satu baris
array_wadah = [int(x) for x in input().split()]


# Coba kita print
for i in range(banyak_baris):
    for j in range(banyak_kolom):
        print(array_wadah[i * banyak_kolom + j], end=" ")
    print()
